.circleci/config.yml

```diff
@@ -0,0 +1,365 @@
+# Templates and Variables
+test_defaults: &test_defaults
+  docker:
+    - image: circleci/node:8.9.4
+  steps:
+    - checkout
+    - restore_cache:
+        keys:
+        - js-dependencies-{{ checksum "package-lock.json" }}
+        - js-dependencies-
+    - attach_workspace:
+        at: /tmp/workspace
+    - run:
+        name: Get JOURNEY_TEST_BASE_URL from Netlify deploy
+        command: echo "export JOURNEY_TEST_BASE_URL=$(cat /tmp/workspace/.netlifyUrl)" >> $BASH_ENV
+    - run: echo "export BUILD_NUMBER=circle-ci-${CIRCLE_BUILD_NUM}" >> $BASH_ENV
+    - run:
+        name: Integration Test
+        command: npm run test:automation:sauce -- --suite ${TEST_SUITE}
+        no_output_timeout: 25m
+    - store_test_results:
+        path: reports/junit
+    - store_artifacts:
+        path: reports
+
+# Main Config
+version: 2
+jobs:
+  install:
+    docker:
+      - image: circleci/node:8.9.4
+    steps:
+      - checkout
+      - run: echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} >> .npmrc
+      - run: npm install
```

Comment:
would using the new npm flag `ci` speed things up a bit here? I've never actually used it

Reply:
It takes about 1:08 to do a full install, right now, which I don't think is bad. The new `ci` flag needs a newer version of `npm` so I didn't put it in here yet.

Usually what I'd be able to do is restore from cache and then do `npm install` which would drop the install time down to about :30, but `npm` has a bug where it sometimes fails the install because it can't delete an old dependency correctly. Figure it's fine this way for now, and we can optimize later.